### PR TITLE
[DOCFIX] Fix `logLevel` command doc

### PR DESCRIPTION
**Pull request by jenoudet**
Represent the `logLevel` command capabilities accurately.

### CODE REVIEW

File: docs/en/operation/User-CLI.md [hunk 1]
```diff
@@ -229,12 +229,13 @@ Status: CANCELED
...
 
-The command follows the format `alluxio logLevel --logName=NAME [--target=<master|workers|job_master|job_workers|host:port>] [--level=LEVEL]`,
+The command follows the format `alluxio logLevel --logName=NAME [--target=<master|workers|job_master|job_workers|host:webPort[:role]>] [--level=LEVEL]`,
 where:
 * `--logName <arg>` indicates the logger's class (e.g. `alluxio.master.file.DefaultFileSystemMaster`)
 * `--target <arg>` lists the Alluxio master or workers to set.
-The target could be of the form `<master|workers|job_master|job_workers|host:webPort>` and multiple targets can be listed as comma-separated entries.
-The `host:webPort` format can only be used when referencing a worker.
+The target could be of the form `<master|workers|job_master|job_workers|host:webPort[:role]>` and multiple targets can be listed as comma-separated entries.
+`role` can be one of `master|worker|job_master|job_worker`. Using the `role` option is useful when an Alluxio process
```
Comment (Xenorith) [inline docs/en/operation/User-CLI.md:237]: before, i could call `--target=master` or `--target=worker:workerWebPort`. with the new role addition, i'm not sure what the format is with when trying to use `role` but instinctively it looks like `targetType[:portIfWorker][:role]. is this accurate? maybe worth giving some examples as opposed to adding more syntax to the existing command

Comment (ZhuTopher) [reply docs/en/operation/User-CLI.md:237]: +1 on providing examples on when/how `:role` is used

Comment (jenoudet) [reply docs/en/operation/User-CLI.md:237]: I added more examples.

**Review by Xenorith — APPROVED** [review]

**Review by ZhuTopher — APPROVED** [review]

**Comment by jenoudet** [comment]
alluxio-bot, merge this please

**Comment by yuzhu** [comment]
alluxio-bot, merge this please

**Pull request merged**